public/scripts/dummy.js

```diff
@@ -0,0 +1,96 @@
+var headers = [
```

Comment:
Why is this file committed..... If ur using thi, it needs to be included in index.html